README.md

```diff
@@ -1,46 +1,180 @@
+### [AWS S3](https://aws.amazon.com/s3/)
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+**Introduction**: Amazon S3 is a object storage to store and retrieve any amount of data.
+
+### [AWS ElastiCache](https://aws.amazon.com/elasticache/)
+
+**Introduction**: Amazon ElastiCache is a fully managed, in-memory caching service with support for Redis and Memcached.
+
+**Details with context**: AWS ElastiCache is being to provide a Redis instance which is being utilized both as a cache and a database for ex.: task status of celery worker machines, caching Athena query IDs, etc.
+
+### [AWS Amplify](https://aws.amazon.com/amplify/)
+
+**Introduction**: Amazon Amplify is a set of purpose-built tools and features that lets frontend web and mobile developers quickly and easily build full-stack applications on AWS, with the flexibility to leverage the breadth of AWS services.
+
+**Details with context**: AWS Amplify is responsible for auto-deploying the frontend codebase without any additional tooling or configuration.
+
+### [AWS EC2](https://aws.amazon.com/ec2/)
+
+**Introduction**: Amazon Elastic Compute Cloud (Amazon EC2) is a web service that provides resizable compute capacity in the cloud.
+
+**Details with context**: AWS EC2 is being used as a virtual machine to run the backend servers. All EC2 machines for Ensembl data lakehouse are running on Ubuntu 22 base image.
+
+<br>
+
+## Achievements & optimisations:
+ - The output files from the python script are structured using **Hive style partitioning** where the top directory denotes the data type, and the sub-directory denotes the species, for example, the S3 URI for the gene of homo_sapiens will be *s3://ensembl-genome-data-parquet/gene/**species=homo_sapiens**/homo_sapiens_core_106_38-gene.parquet*. Hive style partitioning is supported by Athena, which helps to optimize the data querying process by limiting the number of parquet files to be referenced by Athena.
+ - Athena treats each new/duplicate query as a separate request. It does retain the history of query IDs for a period of 45 days so identifying duplicate queries and reusing query IDs from the cache helps to reduce AWS costs as the system scales.
+ - The system can be easily extended for more species and datatype as there are no hard-code dependencies which proves the system is also dynamic in nature and can be easily updated using configuration files to add more datatypes and species in the future.
+ - In the frontend UI, the last successful query ID is cached in local storage and auto-populated on the status and export page.
+
+<br>
+
+## UI snapshots:
+![Home page 1](./ui-snapshots/home_1.png)
+![Home page 2](./ui-snapshots/home_2.png)
+![Home page 3](./ui-snapshots/home_3.png)
+![Home page 4](./ui-snapshots/home_4.png)
+![Status page](./ui-snapshots/status.png)
+![Export page](./ui-snapshots/export.png)
+
```

Comment:
May be update the screen shot with preview displayed.

Reply:
Donr